plugin/src/main/java/org/jenkinsci/plugins/workflow/cps/CpsFlowExecution.java

```diff
@@ -1638,25 +1638,23 @@ public void pause(final boolean v) throws IOException {
     @Restricted(DoNotUse.class)
     @Terminator(attains = FlowExecutionList.EXECUTIONS_SUSPENDED)
     public static void suspendAll() {
-        CpsFlowExecution exec = null;
```

Comment:
Outermost `catch` clause seemed redundant. (`Timeout.close` does not throw exceptions, so it was not for that.)